src/applications/proxy-rewrite/sass/consolidated-patches.scss

```diff
@@ -103,6 +103,10 @@ header.merger {
       .vetnav-panel {
         width: 487px;
 
+        .mm-link-container {
+          width: 220px;
+        }
+
```

Comment:
I assume we are all looking forward to the day we can throw out this TeamSite-specific SCSS.